src/Bindings/ManualBindings.cpp

```diff
@@ -2823,59 +2823,55 @@ class cLuaBlockTracerCallbacks :
 	{
 	}
 
-	virtual bool OnNextBlock(int a_BlockX, int a_BlockY, int a_BlockZ, BLOCKTYPE a_BlockType, NIBBLETYPE a_BlockMeta, eBlockFace a_EntryFace) override
+	virtual bool OnNextBlock(Vector3i a_Block, BLOCKTYPE a_BlockType, NIBBLETYPE a_BlockMeta, eBlockFace a_EntryFace) override
 	{
 		bool res = false;
-		if (!m_Callbacks->CallTableFn(
-			"OnNextBlock",
-			a_BlockX, a_BlockY, a_BlockZ, a_BlockType, a_BlockMeta, a_EntryFace,
-			cLuaState::Return, res
-		))
+		if (
+			!m_Callbacks->CallTableFn("OnNextBlock", a_Block, a_BlockType, a_BlockMeta, a_EntryFace, cLuaState::Return, res) &&
+			!m_Callbacks->CallTableFn("OnNextBlock", a_Block.x, a_Block.y, a_Block.z, a_BlockType, a_BlockMeta, a_EntryFace, cLuaState::Return, res)
+		)
```

Comment:
I need to check how this behaves, if it breaks the current callbacks or not, if it breaks the new callbacks or not

Reply:
it break it now it is changed.